src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+/*  fonts  */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+/*  common   */
+body {
+  font-size: 12px;
+  line-height: 1.2;
+}
```

Comment:
do not apply line-heigth for body and commonly default font-size is applied to html